### PR TITLE
Allow unloaded channels to be enabled

### PR DESCRIPTION
**Pull request by frasercl**
Quick fix: currently, channel settings are not updated on channels that are never loaded. When allen-cell-animated/volume-viewer#173 is published, disabled channels may never be loaded when the user moves to a new time point. Currently, this would cause these channels to be stuck in the disabled state. This change allows the enabled/disabled setting to be changed on unloaded channels.

### CODE REVIEW

File: src/aics-image-viewer/components/App/ChannelUpdater.tsx [hunk 1]
```diff
@@ -31,18 +31,19 @@ const ChannelUpdater: React.FC<ChannelUpdaterProps> = ({ index, channelState, vi
     }, [...deps, image, version]);
   };
 
-  useImageEffect(
```
Comment (ShrimpCryptid) [inline src/aics-image-viewer/components/App/ChannelUpdater.tsx:34]: Quick question because I'm not as familiar with this repo, why the change from `useImageEffect` to a `useEffect` dependent on `image`?

Comment (ShrimpCryptid) [reply src/aics-image-viewer/components/App/ChannelUpdater.tsx:34]: Okay, answered my own question. `useImageEffect` checks for changes to the image but only executes the body if the image is loaded.

**Review by ShrimpCryptid — APPROVED** [review]

**Review by toloudis — APPROVED** [review]
